VIDEO_3_SCRIPT.md

```diff
@@ -0,0 +1,58 @@
+
+It defines a set of files required to describe an API. These files can then be used to create utilities, such as documentation, integration and/or testing tools.
+
+The AsyncAPI Specification doesn't assume any kind of software topology, architecture or pattern. Therefore, a server MAY be a message broker, a web server or any other kind of computer program capable of sending and/or receiving data. However, AsyncAPI offers a mechanism called "bindings" that aims to help with more specific information about the protocol and/or the topology.
+
+## What AsyncAPI brings
+
+Its main contribution to the world of APIs is the standardization of how you can design and describe Event-Driven Architectures in a simplified and unified manner.
+
+We could even say that AsyncAPI is a specification that facilitates, simplifies, and accelerates the work of developers working with APIs. Having defined, documented, and accepted the communication between APIs, AsyncAPI provides an increased and more efficient information control.
+
+AsyncAPI Initiative also develops and maintains an extensive amount of AsyncAPI tools that accelerate application development. The tools' mechanisms for automating complex and essential tasks facilitate code generation, documentation generation, validators, etc. 
+
+AsyncAPI specification is maintained under AsyncAPI Initiative.
+
+## What is AsyncAPI Initiative and what are its objectives
+
+AsyncAPI Initiative is a community that not only maintains the specification. Goal of the initiative is to provide proper tooling support for the spec. That goes from documentation to code generation, from discovery to event management That goes from documentation to code generation, from discovery to event management.
+
+AsyncAPI Initiative currently aims to accomplish three main goals:
+
+1. Reuse other specifications. Most developers complement the event architecture with REST (OpenAPI), GraphQL, and RPC APIs. Let us embrace reusability! you can change to Most developers complement the event architecture with REST (OpenAPI), GraphQL, or Avro and CloudEvents. Let us embrace reusability!
+
+2. Make the development experience super seamless, from idea to production. Users should be able to create their first API in a matter of minutes without prior knowledge of AsyncAPI.
+
+3. Reach the first position among the most used API specifications. This will require a growth in the number of community members by ensuring it is a community-driven project running under open governance.
+
+## AsyncAPI joins Linux Foundation
+
+An important step the initiative took towards its consolidation was to partner with Postman at the end of 2020. This partnership allows investment in a dedicated department working full time on AsyncAPI spec, tools and community-building aspects.
```

Comment:
**An important step the AsyncAPI initiative took for its consolidation, was to partner with Postman at the end of 2020. Thanks to this partnership, a dedicated department works full-time on the AsyncAPI spec, its tools, and community-building aspects.**